Celeste.Mod.mm/Content/Dialog/English.txt

```diff
@@ -22,6 +22,11 @@
 	POSTCARD_LEVELGONE= 		{big}Thank you ((player))!{/big}{n}But our {#ff1144}((sid)){#} is on another mountain.
 	POSTCARD_LEVELLOADFAILED= 	{big}Sorry!{/big}{n}But {#ff1144}((sid)){#} has failed loading.{n}Please send your {#44adf7}log.txt{#}{n}to the map builder.
 	POSTCARD_LEVELNOSPAWN= 		{big}Oops!{/big}{n}Someone misplaced your {#44adf7}starting point!{#}{n}Please check if your map{n}contains a {#44adf7}Player{#} entity.
+	POSTCARD_BADTILEID=			{big}Oops!{/big}{n}There is an undefined ((type))tile id {#ff1144}((id)){#} at{n}{#ff1144}(((x)), ((y))){#} in room {#ff1144}((room)){#}{n}Check your {#44adf7}log.txt{#} for more info.
```

Comment:
```suggestion
	POSTCARD_BADTILEID=			{big}Oops!{/big}{n}There is an undefined ((type))tile id {#ff1144}((id)){#} at{n}{#ff1144}(((x)), ((y))){#} in room {#ff1144}((room)){#}.{n}Check your {#44adf7}log.txt{#} for more info.
```

Reply:
The lack of periods is actually intentional lol. You'll notice they're all after the variable names since those can potentially be very long. If they're too long, the period will print on the next line by itself which looks way worse than just not having a period after an SID